src/network/local_socket_reply.cpp

```diff
@@ -170,6 +187,10 @@ void mp::LocalSocketReply::send_request(const QNetworkRequest& request, QIODevic
         http_data += "User-Agent: " + user_agent + "\r\n";
     }
 
+    // Workaround weird issue in LXD's use of Go where Go's HTTP handler thinks there is more
+    // data and "sees" and "empty" query and reponds with an unexpected 400 error
+    http_data += "Connection: close\r\n";
```

Comment:
Is there a LXD bug we could reference?

Reply:
@stgraber dug in to this and found it's not a LXD bug, but either a bug in Go or some weird expected behavior in it.  I don't know if he opened a bug anywhere for this and felt it wasn't necessary to follow up on either since we have a way to avoid it.